.rustfmt.toml

```diff
@@ -0,0 +1,27 @@
+max_width = 100
+hard_tabs = false
+tab_spaces = 4
+newline_style = "Auto"
+use_small_heuristics = "Default"
+fn_call_width = 60
+attr_fn_like_width = 70
+struct_lit_width = 18
+struct_variant_width = 35
+array_width = 60
+chain_width = 60
+single_line_if_else_max_width = 50
+single_line_let_else_max_width = 50
+reorder_imports = true
+reorder_modules = true
+remove_nested_parens = true
+short_array_element_width_threshold = 10
+match_arm_leading_pipes = "Never"
+fn_params_layout = "Tall"
+match_block_trailing_comma = false
+edition = "2024"
+style_edition = "2024"
+merge_derives = true
+use_try_shorthand = false
```

Comment:
why `false`?

Reply:
I generated the file with `cargo fmt -- --print-config current . --edition 2024 > .rustfmt.toml` so I think that's the default.

The parameter in question: https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#use_try_shorthand.

We don't use that macro anywhere afaik but I don't see why not change it to `true` since `?` is the recommended syntax now

Reply:
Smart.
Yeah, I don't see why it shouldn't be changed to `?`. It sounds like `false` makes no change either way so I guess it's the default so you have to opt-in to making this change.